torch/nn/modules/loss.py

```diff
@@ -1070,8 +1070,8 @@ class CrossEntropyLoss(_WeightedLoss):
                 \text{if reduction} = \text{`sum'.}
             \end{cases}
 
-      Note that this case is equivalent to the combination of :class:`~torch.nn.LogSoftmax` and
-      :class:`~torch.nn.NLLLoss`.
+      Note that this case is equivalent to a :class:`~torch.nn.LogSoftmax`
+      passed into a :class:`~torch.nn.NLLLoss`.
```

Comment:
Maybe its slightly clearer if we reword to something like: 
```
Note that this case is equivalent to applying :class:`~torch.nn.LogSoftmax` on input, followed by :class:`~torch.nn.NLLLoss`. 
```

Reply:
Good suggestion!  Implemented it